web/src/transmission.js

```diff
@@ -41,6 +41,24 @@ export class Transmission extends EventTarget {
 
     // Initialize the implementation fields
     this.filterText = '';
+    this.searchOperators = [
```

Comment:
I have some kind of fear on performance loss by moving the constant vars into local scope so it's going to be computed each time the function is called. Maybe that's not the case?

Reply:
Well... There will be performance loss, but it will be negligible. None of the values in `searchOperators` are computed after all, they are just constants.

Reply:
Sure, sure